README.md

```diff
@@ -111,7 +111,7 @@ NSString *const username = @"Skroob";
 NSString *const myLuggageCombination = [myValet stringForKey:username];
 ```
 
-In addition to allowing the storage of strings, Valet allows the storage of `Data` objects via `setObject(_ object: Data, forKey key: Key)` and `-objectForKey:`. Valets created with a different class type, via a different initializer, or with a different accessibility attribute will not be able to read or modify values in `myValet`.
```

Comment:
There was no reason for this paragraph to discuss Objective-C syntax. It was a holdover from when we switched to Swift.